packages/rum-angular/test/e2e/with-router/app.e2e-spec.js

```diff
@@ -26,29 +26,50 @@
 const { waitForApmServerCalls } = require('../../../../../dev-utils/webdriver')
 
 describe('Angular router integration', function() {
-  beforeAll(() => browser.url('/test/e2e/with-router/'))
+  /**
+   * Change ELEMENT_KEY to approprioate value when using `devtools`
+   * automation protocol
+   * https://github.com/webdriverio/webdriverio/blob/e942ce4d802161ac12579553889d9068dccf317c/packages/devtools/src/constants.ts#L8
+   */
+  const ELEMENT_KEY = 'ELEMENT'
+  beforeAll(async () => {
+    await browser.url('test/e2e/with-router/build/')
+  })
 
-  it('should run angular app and capture route-change', function() {
+  it('should run angular app and capture route-change', async () => {
     /**
      * Should render home page on load
      */
-    const notFoundElement = $('app-root app-home h2')
-    expect(notFoundElement.getText()).toEqual('Home page')
+    const result = await browser.findElement(
```

Comment:
the code on this file is converted to make sure it works for devtools automation protocol as well.